app/assets/stylesheets/theme/custom.css.scss

```diff
@@ -89,4 +89,13 @@ body {
     padding-right: 0;
     padding-left: 0;
   }
+}
+
+/* Temp css */
+.block {
+  overflow: hidden;
+}
+
+#search_query {
```

Comment:
Avoid using id selectors<br>Selector `search_query` should be written in lowercase with hyphens